sites/docker-compose.cookhamwood.yml

```diff
@@ -0,0 +1,58 @@
+version: "3"
+
+services:
+  hub-be:
+    environment:
+      HUB_DB_ENV_MYSQL_DATABASE: hubdb
+      HUB_DB_ENV_MYSQL_USER: hubdb_user_2y478gt
+      HUB_DB_PORT_3306_TCP_ADDR: hub-db
+      HUB_EXT_FILE_URL: https://digital-hub.cwi.dpn.gov.uk/sites/default/files
+      PHP_MEMORY_LIMIT: 256M
+      PHP_UPLOAD_MAX_FILE_SIZE: 500M
+      PHP_POST_MAX_SIZE: 500M
+      PIWIK_URI: //digital-hub.cwi.dpn.gov.uk:12001
+
+    volumes:
+      - /content/moj_dhub_prod001_app/usr/share/nginx/html/moj_be/sites/default/files:/var/www/html/sites/default/files
+
+  hub-db:
+    volumes:
+      - /data/hub-db:/var/lib/mysql:rw
+
+  hub-elasticsearch:
+    volumes:
+      - /data/esdata:/usr/share/elasticsearch/data:rw
+
+  hub-matomo:
```

Comment:
do we need the matomo in the docker-compose now?

Reply:
No, not really, we could remove from all of them, various files that are no longer required too I think :)